src/module-2/components/stylesheets/global.css

```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;900&display=swap');
+
+* {
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-size: 6.25%;
```

Comment:
minor: base font size of 10px may be easier for the further calculations with relative units -  [mdn](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Fundamentals)